redi-connect-front/src/components/organisms/LoggedOutNavbar.scss

```diff
@@ -0,0 +1,46 @@
+@import "~bulma/sass/utilities/_all";
+@import "../../styles/_variables.scss";
+
+.navbar {
+  &__wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    height: 4.5rem;
+
+    &-mobile {
+      position: absolute;
+      width: 100vw;
+      height: 100vh;
+      z-index: 10;
+      left: 0;
+      top: 0;
+      padding: 8rem 1rem;
+      background-color: $white;
```

Comment:
A shorthand notation could be nicer to read. Check it out, maybe only as an info.

Reply:
This is just a placeholder for now since we don't know how the Navbar should look like if we are logged out atm.